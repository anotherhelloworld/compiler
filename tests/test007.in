123 321
9854e-23-567