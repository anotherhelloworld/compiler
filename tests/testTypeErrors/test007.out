Incompatible types: expected: "integer" but "real" found Line 9 Column 11
